pkg/log/zap/zap.go

```diff
@@ -145,6 +146,10 @@ type Options struct {
 	// ZapOpts allows passing arbitrary zap.Options to configure on the
 	// underlying Zap logger.
 	ZapOpts []zap.Option
+
+	// Scheme for deducing GVK for arbitrary runtime.Object if not
+	// explicitly specified
+	Scheme *runtime.Scheme
```

Comment:
It would be nice to default this in `addDefaults`

Reply:
Something like:
```go
import "	"k8s.io/client-go/kubernetes/scheme"

...
	if o.Scheme == nil {
		o.Scheme = scheme.Scheme
	}
```

Or did you mean something else?

Reply:
Exactly that

Reply:
done